pkg/config/deployment_terraform_test.go

```diff
@@ -90,6 +90,7 @@ func TestTerraformDeploymentConfig(t *testing.T) {
 								Name: model.StageWaitApproval,
 								WaitApprovalStageOptions: &WaitApprovalStageOptions{
 									Approvers: []string{"foo", "bar"},
+									Timeout:   defaultWaitApprovalTimeout, // set default timeout value when nothing sets on config
```

Comment:
nit
```go
WaitApprovalStageOptions: &WaitApprovalStageOptions{
    Approvers: []string{"foo", "bar"},
    // Use defaultWaitApprovalTimeout on unset timeout value for WaitApprovalStage.
    Timeout: defaultWaitApprovalTimeout,
```
or remove this comment and place it to implement source at [L137](https://github.com/pipe-cd/pipe/pull/1386/files#diff-3c6bc697f23721a03c348b45edf829c1521cb4bc7506a3e7b137aecc85466d1dR137)